libindy/src/services/crypto/mod.rs

```diff
@@ -425,6 +452,183 @@ impl CryptoService {
 
         Ok(res)
     }
+
```

Comment:
This through line 631 doesn't seem like it would fit in the crypto_service now, but I'm not exactly sure where the best place to put them would be. for the recipient focused functions, I plan to move those into the command layer which shouldn't be a problem. For the other two symmetrical crypto encrypt/decrypt functions though these don't feel like they fit, but I don't know a better place to put them. Any suggestions? @jovfer @vimmerru